src/execution_plan/execution_plan.c

```diff
@@ -230,6 +230,20 @@ OpBase* ExecutionPlan_LocateOp(OpBase *root, OPType type) {
     return NULL;
 }
 
+void ExecutionPlan_Taps(OpBase *root, OpBase ***taps) {
+    if(root == NULL) return;
+    int t = OPType_ALL_NODE_SCAN |
+            OPType_NODE_BY_LABEL_SCAN |
+            OPType_INDEX_SCAN |
+            OPType_NODE_BY_ID_SEEK;
+
+    if(root->type & t) *taps = array_append(*taps, root);
```

Comment:
Why not just extend `OP_SCAN`? (Or if that could cause an inconsistency like trying to replace an ID seek with an ID seek, `OP_SCAN | OPType_NODE_BY_ID_SEEK`).

Reply:
👍